core/src/main/java/hudson/model/UnprotectedRootAction.java

```diff
@@ -32,4 +32,26 @@
  * @since 1.398
  */
 public interface UnprotectedRootAction extends RootAction, ExtensionPoint {
+
+    /**
+     * Gets the URL path name.
+     *
+     * <p>
+     * For example, if this method returns "xyz", and if the parent object
+     * (that this action is associated with) is bound to /foo/bar/zot,
+     * then this action object will be exposed to /foo/bar/zot/xyz.
+     *
+     * <p>
+     * This method should return a string that's unique among other {@link Action}s.
+     *
+     * <p>
+     * <strong>Unlike normal {@link Action}s the returned string is always assumed to be relative to the context path of the Jenkins webapp and should not be prefixed with a '/'.</strong>
```

Comment:
It is inconsistent with the *specified* behavior where you can start an action with '/' to have it relative to the root.
> If the returned string starts with '/', like '/foo', then it's assumed to be relative to the context path of the Jenkins webapp.

Or are you we saying that the `Action` javadoc is just garbage and no action should ever start with a '/' and if you want something at the root you should always use `RootAction` or a subtype?